graphene_plugin.py

```diff
@@ -267,13 +291,10 @@ def get_type_mismatch_error_message(arg_name: str, *, graphene_type: Type, resol
 class GraphenePlugin(Plugin):
     @staticmethod
     def get_base_class_hook(fullname: str) -> Optional[Callable[[ClassDefContext], None]]:
+        @strict_optional_set(True)
```

Comment:
The effects of not doing this were driving me _crazy_. `is_equivalent(builtins.int, Union[builtins.int, None])` was returning `True` and I was like ffffffuuuuuuuuu.

Reply:
Wtf? This isn't specified in the test's mypy.ini, and I would assume this would be defaulting to `False` except on very old versions of mypy

![image](https://user-images.githubusercontent.com/7363269/75922753-aa317d80-5e31-11ea-990d-b7a5c55e2a50.png)